utils/prompt.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// confirm prompts a user for a confirmation with message
+// and returns true with no error if input is "yes" or "y" (case-insensitive),
+// otherwise false.
+func Confirm(message string) (bool, error) {
+	fmt.Print(message)
+	yes := []string{"yes", "y"}
```

Comment:
nit: I would prefer to call something like this `validOptions`, and then below instead of `for_, valid := range yes` I think it could be a little easier to read `for _, validOption := range validOptions`? Let me know what you think, not a blocker.
```suggestion
	validOptions := []string{"yes", "y"}
```